src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const res = {};
+
+  const estilos = sourceString.split(';').filter(checkEmpty);
```

Comment:
The `split(';')` method is used correctly to separate the string into parts. However, ensure that the input string format is always consistent with this expectation, as any deviation might cause unexpected results.